content/cloud/variables.md

```diff
@@ -0,0 +1,208 @@
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin templates install --git https://github.com/fermyon/spin-python-sdk --upgrade
+$ spin plugins install py2wasm --yes
+```
+
+The output from the command above will be similar to the following:
+
+<!-- @nocpy -->
+
+```text
+Copying remote template source
+Installing template http-py...
+Installed 1 template(s)
+
++---------------------------------------------+
+| Name      Description                       |
++=============================================+
+| http-py   HTTP request handler using Python |
++---------------------------------------------+
+
+Plugin 'py2wasm' was installed successfully!
+
+Description:
+	A plugin to convert Python applications to Spin compatible modules
+
+Homepage:
+	https://github.com/fermyon/spin-python-sdk
+```
+
+## Creating Our Spin Application
+
+Let's start by [creating a new Spin application](/cloud/cli-reference#new) from a the Python template. Using the docs as a reference, we can perform the following:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin new http-py pw_checker
+Project description: A Spin app that validates a password
+HTTP base: /
+HTTP path: /...
+```
+
+## Adding Variables to an Application Component
+
+Next, let's add two variables to our application. Add a top level `[variables]` section to the application manifest (`spin.toml`) and define a `password` and `expected_password` variable, marking them as required. Now, add these variables to a component by referencing them in the `[component.config]` section in the `pw-checker` component. Instead of statically assigning the value of the config variables, we are referencing the dynamic variables with [mustache](https://mustache.github.io/)-inspired string templates. The resultant application manifest should look similar to the following:
+
+> Note: only components that explicitly use the variables in their configuration section will get access to them.
+
+<!-- @selectiveCpy -->
+```toml
+spin_manifest_version = "1"
+description = "A Spin app with a dynamically updatable "
+name = "pw_checker"
+trigger = { type = "http", base = "/" }
+version = "0.1.0"
+
+[variables]
+expected_password = { required = true }
+password = { required = true }
+
+[[component]]
+id = "pw-checker"
+source = "app.wasm"
+[component.trigger]
+route = "/..."
+[component.build]
+command = "spin py2wasm app -o app.wasm"
+[component.config]
+password = "{{ password }}"
+expected_password = "{{ expected_password }}"
+```
+
+## Using Variables in a Spin Application
+
+Now that we have defined our variables and exposed them to our component, lets implement our Spin application. The application should retrieve each variable and validate that the expected password matched the actual password. We will use the Spin `config_config` SDK to get the values. 
```

Comment:
typo

```suggestion
Now that we have defined our variables and exposed them to our component, let's implement our Spin application. The application should retrieve each variable and validate that the expected password matched the actual password. We will use the Spin `config_config` SDK to get the values. 
```

Reply:
Also I think "the Spin `config_config` SDK" might be wrong... "the `spin_config` module" perhaps?  (or "the `spin_config` module from the Spin SDK"?)  (or is it "package" rather than module, not sure)